src/ng/templateRequest.js

```diff
@@ -45,12 +79,10 @@ function $TemplateRequestProvider() {
         transformResponse = null;
       }
 
-      var httpOptions = {
-        cache: $templateCache,
-        transformResponse: transformResponse
-      };
-
-      return $http.get(tpl, httpOptions)
+      return $http.get(tpl, extend({
+          cache: $templateCache,
+          transformResponse: transformResponse
+        }, httpOptions))
```

Comment:
Shouldn't it be other way around? I mean, we should always enforce `cache: $templateCache` and certain transform, no?